kitsune/upload/tasks.py

```diff
@@ -1,23 +1,21 @@
+import io
 import logging
 import os
-import io
 import subprocess
 from tempfile import NamedTemporaryFile
 
+from celery import task
 from django.conf import settings
 from django.core.files.base import ContentFile
 from django.core.files.storage import default_storage
-
-from celery import task
 from PIL import Image
 
-
 log = logging.getLogger('k.task')
 
 
 @task(rate_limit='15/m')
 def generate_thumbnail(for_obj, from_field, to_field,
-                       max_size=settings.THUMBNAIL_SIZE):
+                       max_size=settings.THUMBNAIL_SIZE, serializer="pickle"):
```

Comment:
Fair enough, could we pass a path to the image instead?

Reply:
Worth investigating